src/core/content/util.js

```diff
@@ -327,7 +327,7 @@ const Util = {
 	 * a first element with the attribute available.
 	 *
 	 * @param  {String|Array} selectors Single selector or array of selectors
-	 * @param  {String} attr Attrubute to get
+	 * @param  {String} attr Attribute to get
```

Comment:
Ideally these other changes should go into seperate pull requests so we can track changes independently.

Reply:
I will push a new merge for this typo :)